src/components/Blog/CustomRichBlocks.tsx

```diff
@@ -55,8 +88,43 @@ export const CustomRichBlocks = ({
         } catch (error) {
           return;
         }
-      } else if (children[0].props.text.includes('<WIDGET>')) {
-        return <Widget starterVariant="default" />;
+      } else if (children[0].props.text.includes('<WIDGET')) {
```

Comment:
Would be good to put the related if `props.text` part into the big if text, logic of the different type of components could be splitted to different files.

e.g.
``` 
import RichBlockCTA from '@/components/RchBlocks/RchBlockCTA'
import RichBlockWidget from '@/components/RchBlocks/RchBlockWidget'
//...
```

Reply:
Let´s sync up on this one. I am afraid I don´t get it.

Reply:
+1